packages/shared/lib/contexts/onboarding/errors/stronghold-backup.errors.ts

```diff
@@ -11,9 +11,11 @@ export class UnsupportedBackupFileError extends BaseError {
 }
 
 export class CannotRestoreWithMismatchedCoinTypeError extends BaseError {
-    constructor() {
+    constructor(isClaiming: boolean = false) {
         super({
-            message: localize('notifications.restoreFromStrongholdBackup.wrongProtocolForClaiming'),
+            message: isClaiming
+                ? localize('notifications.restoreFromStrongholdBackup.wrongProtocolForClaiming')
+                : localize('notifications.restoreFromStrongholdBackup.wrongProtocol'),
             localizeMessage: false,
             showNotification: true,
```

Comment:
We should use the `DEFAULT_APP_ERROR_PARAMETERS`:

```suggestion
            ...DEFAULT_APP_ERROR_PARAMETERS,
```